static/js/saved-query.js

```diff
@@ -281,7 +282,7 @@ $(document).ready(function () {
    {
      field: "filterTab",
      headerName: "FilterTab",
-     hide: true,
+     hide: false,
```

Comment:
1. Don't need to show the filter tab on the saved query page.
2. The Filter tab is not getting maintained when you refresh.
For example, if the query mode (default) is set to code, and you run a query through the builder and then refresh the tab, it goes to code mode. It should remain on the builder tab.